From 03653a7d75ba8b24e28bc3adf5348ea3d0faf2b0 Mon Sep 17 00:00:00 2001
From: Michael Palimaka <kensington@gentoo.org>
Date: Thu, 16 May 2019 19:39:44 +1000
Subject: [PATCH] Port knsbackendtest to ecm_add_test

Reviewers: #plasma, apol

Reviewed By: apol

Subscribers: asturmlechner, plasma-devel

Tags: #plasma

Differential Revision: https://phabricator.kde.org/D21239
---
 libdiscover/backends/KNSBackend/tests/CMakeLists.txt | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/libdiscover/backends/KNSBackend/tests/CMakeLists.txt b/libdiscover/backends/KNSBackend/tests/CMakeLists.txt
index 4fabe5ec..e64eb8ca 100644
--- a/libdiscover/backends/KNSBackend/tests/CMakeLists.txt
+++ b/libdiscover/backends/KNSBackend/tests/CMakeLists.txt
@@ -1,11 +1,3 @@
 include_directories(..)
 
-add_executable(knsbackendtest KNSBackendTest.cpp)
-add_test(knsbackendtest knsbackendtest)
-ecm_mark_as_test(knsbackendtest)
-target_link_libraries(knsbackendtest
-    Discover::Common
-    
-    Qt5::Test Qt5::Core
-    KF5::NewStuff KF5::Attica
-)
+ecm_add_test(KNSBackendTest.cpp TEST_NAME knsbackendtest LINK_LIBRARIES Discover::Common Qt5::Core Qt5::Test KF5::Attica KF5::NewStuff)
-- 
2.21.0

